triggers/service/auth/src/main/scala/com/daml/oauth/server/Config.scala

```diff
@@ -16,7 +18,9 @@ case class Config(
     // Secret used to sign JWTs
     jwtSecret: String,
     // Only authorize requests for these parties, if set.
-    parties: Option[Set[Party]]
+    parties: Option[Set[Party]],
+    // Use the provided clock instead of system time for token generation. For testing only.
+    clock: Option[Clock],
```

Comment:
Isn’t the whole oauth server only for testing?

Reply:
Indeed, I've removed the comment.